static/app/components/feedback/feedbackOnboarding/sidebar.tsx

```diff
@@ -288,6 +290,9 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
   }
 
   function getConfig() {
+    if (webApiPlatform) {
+      return 'feedbackOnboardingWebApi';
+    }
```

Comment:
Is it necessary to play this via the docs mechanism?
IMHO you could simply hardcode to display the banner for the web api platforms as the content is not platform specific.
Would keep it simple as long as possible.

Reply:
makes sense to me! i'll update to remove the new config